libs/sdk-core/src/breez_services.rs

```diff
@@ -451,9 +451,10 @@ impl BreezServices {
         filter: PaymentTypeFilter,
         from_timestamp: Option<i64>,
         to_timestamp: Option<i64>,
+        amount_of_payments: Option<u32>,
```

Comment:
In order to get the support not only in rust we need to change the udl file and `BlockingBreezServices` in the sdk-binding crate.